modules/nomad-cloudinit-ubuntu-v1/templates/nomad_user_data.tpl

```diff
@@ -24,6 +24,14 @@ apt-get install -y "linux-image-$UNAME"
 apt-get update
 apt-get -y install docker-ce=17.03.2~ce-0~ubuntu-xenial
 
+# force docker to use userns-remap to mitigate CVE 2019-5736
+apt-get -y install jq
+mkdir -p /etc/docker
+[ -f /etc/docker/daemon.json ] || echo '{}' > /etc/docker/daemon.json
+tmp=$(mktemp)
+cp /etc/docker/daemon.json /etc/docker/daemon.json.orig
+jq '."userns-remap"= "default"' /etc/docker/daemon.json > "$tmp" && mv "$tmp" /etc/docker/daemon.json
```

Comment:
This actually throws an error on Trusty using jq 1.3. It does work in 1.5.1 but not 1.3, the latest available to Trusty. Adding some brackets solves it:

```suggestion
jq '.["userns-remap"]="default"' /etc/docker/daemon.json > "$tmp" && mv "$tmp" /etc/docker/daemon.json
```